client/flutter/lib/l10n/intl_es.arb

```diff
@@ -1,23 +1,26 @@
 {
-    "worldHealthOrganization": "",
-    "learnMore": "",
-    "protectYourself": "",
-    "checkYourHealth": "",
-    "localMaps": "",
-    "shareTheApp": "",
-    "healthCheck": "",
-    "feelingDistressed": "",
-    "washHands": "",
-    "cougningAndSneezing": "",
-    "throwAwayTissue": "",
-    "washHandsFrequently": "",
-    "socialDistancing": "",
-    "seekMedicalCare": "",
-    "infoForEveryone":"",
-    "infoForParents":"",
-    "childStress":"",
-    "childLove":"",
-    "childDistance":"",
-    "childRoutines":"",
-    "childFacts":""
-  }
\ No newline at end of file
+  "worldHealthOrganization": "Organización Mundial de la Salud",
+  "learnMore": "Aprenda más",
+  "protectYourself": "Protéjase",
+  "checkYourHealth": "Consulte su estado de salud",
+  "localMaps": "Mapas Locales",
+  "shareTheApp": "Compartir la aplicación",
+  "healthCheck": "Consulta de salud",
+  "feelingDistressed": "Se siente angustiado/a?",
+  "washHands": "Lávese las manos a menudo con agua y jabón",
+  "cougningAndSneezing": "Cuando tosa o estornude, tápese la boca y la nariz con un pañuelo, o con su codo",
+  "throwAwayTissue": "Eche el pañuelo a una papelera con tapa inmediatamente después de su uso",
+  "washHandsFrequently": "Lávese las manos frecuentemente",
+  "socialDistancing": "Evite el contacto cercano con personas, y mantenga una distancia de separación prudencial",
+  "seekMedicalCare": "Busque asistencia sanitaria si tiene fiebre, tos, o dificultad al respirar",
+  "infoForEveryone": "Información para Todos/as",
+  "infoForParents": "Información para Padres y Madres",
+  "childStress": "Los niños responden al estrés de forma muy diferente, estando más ansiosos, agitados, mojando la cama...\nResponde a los niños de forma que se sientan protegidos, escucha sus preocupaciones, y dales un extra de amor y atención",
+  "childLove": "Los niños necesitan la atención y el amor de los adultos en los momentos más difíciles. Dedícales un extra de tiempo y atención.\nNo olvides escucharles, hablarles amablemente y de forma tranquilizadora.\nSi es posible, dales la oportunidad de jugar y poder relajarse.",
+  "childDistance": "Intenta mantener a los niños tan cerca como puedas de sus padres, madres y cuidadores, tanto como sea posible. Si se produce cualquier tipo de separación (por ejemplo, por hospitalización), asegúrate the mantener contacto regular (por ejemplo, por teléfono) y proporcionar alivio",
```

Comment:
In the rest of the translation you´re using 2nd person imperative **lávese, evite, aprenda** but in **asegúrate** you are using 1st person, I think you shuould use **asegúrese**

Reply:
Same thing with **Intenta**, should be **Intente**